tests/testColorterms.py

```diff
@@ -0,0 +1,55 @@
+import os
+import numbers
+import unittest
+
+import lsst.utils.tests as utilsTests
+import lsst.pipe.tasks.photoCal as photoCal
+
+class ColortermOverrideTestCase(unittest.TestCase):
+    """Test that colorterms specific to CFHT override correctly"""
+    def setUp(self):
+        colortermsFile = os.path.join(os.environ["OBS_CFHT_DIR"],"config","colorterms.py")
+        self.photoCalConf = photoCal.PhotoCalConfig()
+        self.photoCalConf.colorterms.load(colortermsFile)
+
+    def testColorterms(self):
+        """Test that the colorterm libraries are formatted correctly"""
+        refBands = ["u", "g", "r", "i", "z"]
+        cfhtBands = ["u","g","r","i","z"]
```

Comment:
Also, why have both refBands and cfhtBands? If both are needed (because one is modified later), and if both are supposed to be equal, why not make this clear with `cfhtBands = refBands[:]`